src/PrestaShopBundle/Controller/Admin/Improve/International/TaxController.php

```diff
@@ -192,9 +192,9 @@ public function editAction(Request $request, $taxId)
         $taxFormHandler = $this->get('prestashop.core.form.identifiable_object.handler.tax_form_handler');
         $taxFormBuilder = $this->get('prestashop.core.form.identifiable_object.builder.tax_form_builder');
 
+        $taxForm = $taxFormBuilder->getFormFor((int) $taxId);
```

Comment:
Again the warning about hook and try/catch (and probably more 😋 I will stop mentioning it)